coveralls/cli.py

```diff
@@ -60,7 +62,8 @@ def main(argv=None):
     try:
         coverallz = Coveralls(token_required,
                               config_file=options['--rcfile'],
-                              service_name=options['--service'])
+                              service_name=options['--service'],
+                              base_dir=options.get('--base_dir', None))
```

Comment:
To keep up with the below null vs string comments:
```suggestion
                              base_dir=options.get('--base_dir') or '')
```